### PR TITLE
Pinned flows

### PR DESCRIPTION
**Pull request by kurund**

### CODE REVIEW

**Comment by kurund** [comment]
@mdshamoon 

Can you please review this? I feel one cypress test will fail and needs to be fixed once this is merged.

File: src/common/constants.ts [hunk 1]
```diff
@@ -141,7 +141,7 @@ export const setColumnToBackendTerms: any = (listName: string, columnName: strin
   };
 
   const additionalItems: any = {
-    flow: { TITLE: 'name' },
+    flow: { TITLE: 'name', ' ': 'is_pinned' },
```
Comment (mdshamoon) [inline src/common/constants.ts:144]: This does not feel right somehow. We should think about fixing this sometime soon. Creating an issue for this https://github.com/glific/glific-frontend/issues/2097

File: src/mocks/Flow.tsx [hunk 2]
```diff
@@ -190,8 +191,8 @@ export const filterFlowNewQuery = {
       opts: {
         limit: 50,
         offset: 0,
-        order: 'ASC',
-        orderWith: 'name',
+        order: 'DESC',
+        orderWith: ' ',
```
Comment (mdshamoon) [inline src/mocks/Flow.tsx:195]: The order_with should be is_pinned I think. Let me know if I am missing something here.

File: src/common/constants.ts (hunk 1 above)
Comment (kurund) [reply src/common/constants.ts:144]: Yes, it is weird :) We map using what's displayed in the UI which is not the best way. This is also an issue when it comes to localising it.  So this is only a workaround. I will take a look at clean-up once done with package stuff.

File: src/mocks/Flow.tsx (hunk 2 above)
Comment (kurund) [reply src/mocks/Flow.tsx]: Yes, thanks for the fix.

**Pull request merged**